pkgs/development/tools/misc/segger-jlink/default.nix

```diff
@@ -0,0 +1,136 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, qt4
+, udev
+, system
+, config
+, acceptLicense ? config.segger-jlink.acceptLicense or false
+}:
+
+let
+  supported = {
+    x86_64-linux = {
+      name = "x86_64";
+      sha256 = "0cmd9ny6mslj9260scply573gbmn6k5r9wyxmcbjb14k4b3sldic";
+    };
+    i686-linux = {
+      name = "i386";
+      sha256 = "1jjh5a8y17ma369s9k0xlyr8i10v2r1kxmv0i9v4cix5dnjyq0pp";
+    };
+    aarch64-linux = {
+      name = "arm64";
+      sha256 = "0p3gjv36a82xyps20drm7h0vm0jmz5gd930ynwr3iy46md41hgyv";
+    };
+    armv7l-linux = {
+      name = "arm";
+      sha256 = "0z9q3mqhfwb341mk5p1d3vp6mfwffcdk68968jy1qr2ga2s34wpb";
+    };
+  };
+
+  platform = supported.${system} or (throw "unsupported platform ${system}");
+
+  version = "750";
+
+  url = "https://www.segger.com/downloads/jlink/JLink_Linux_V${version}_${platform.name}.tgz";
+
+  throwLicense = throw ''
```

Comment:
```suggestion
  assert !acceptLicense -> throw ''
```

Reply:
Thank you, added it in an extra commit for now. Will squash once we found a good way to move `JLink` folder.